src/script/lua_api/l_object.cpp

```diff
@@ -738,6 +738,31 @@ int ObjectRef::l_get_attach(lua_State *L)
 	return 5;
 }
 
+// get_children(self)
+int ObjectRef::l_get_children(lua_State *L)
+{
+	GET_ENV_PTR;
+
+	ObjectRef *ref = checkobject(L, 1);
+	ServerActiveObject *sao = getobject(ref);
+	if (sao == NULL)
+		return 0;
+
+	//Do it
+	const std::unordered_set<int> child_ids = sao->getAttachmentChildIds();
+	if (child_ids.empty())
```

Comment:
```suggestion
```
I think it would be better to pass a table in any case even if it is empty instead nil. You would not need to append `or {}` to a statement as in your testing example everytime when you want to retrieve it. Just it makes no sense.